release/README.md

```diff
@@ -33,6 +33,8 @@ https://semver.org/
 
     When a released is published the public [user-facing release notes](https://github.com/elastisys/compliantkubernetes/blob/main/docs/release-notes/ck8s.md) needs to be updated. The new release needs to be added and the list can be trimmed down to only include the supported versions.
 
+    Create two new branches `release-vXY.Z` and `<personal-tag>/release-vXY.Z`. Add the release notes to `<personal-tag>/release-vXY.Z` and then create two PRs. One to `release-vXY.Z` and one to `main`.
+
     Add bullet points of major changes within the cluster that affects the user as defined [here](https://compliantkubernetes.io/user-guide/). This includes any change within the cluster that may impact the user experience, for example new or updated feature, or the deprecation of features.
```

Comment:
@aarnq I think this part is now broken and should reference something else.

Reply:
OK, it seems like the reference is completely gone so you'll have to remove it for now.